docs/debugging-testing/testing/unit_testing.md

```diff
@@ -227,40 +202,55 @@ With the following steps, you can write a unit test. This example creates dummy
     }
     ```
 
-1. Stub all external dependencies. Create the following stub in the Mbed OS root directory `UNITTESTS/stubs`:
-
-    **OtherClass_stub.cpp**
-
+    ### CMake input source file for UUT
+    ```
+    set(TEST_NAME myclass-unittest)
+
+    add_executable(${TEST_NAME})
+  
+    target_sources(${TEST_NAME}
+        PRIVATE
+            ${MBED_PATH}/example/tests/unittests/MyClass/MyClass.cpp
+            test_MyClass.cpp
+    )
+    
+    target_link_libraries(${TEST_NAME}
+        PRIVATE
+            mbed-headers
+            mbed-stubs            
+            gmock_main
+    )
+    
+    add_test(NAME "${TEST_NAME}" COMMAND ${TEST_NAME})
```

Comment:
As we called "add_executable(${TEST_NAME})" so it will create an executable bin file with this ${TEST_NAME} name and specifying the executable binary file to COMMAND for running that binary by CTest.